cmd/releasego/templates/announcement.template.html

```diff
@@ -0,0 +1,11 @@
+<p>The Microsoft builds of the Go security patches released today, {{ .ReleaseDate }}, are now <a href="https://github.com/microsoft/go#binary-distribution">available for download</a>. The new versions are:</p>
+    <ul>
+        {{- range .MSGoVersions }}
+        <li>{{ . }}</li>
+        {{- end }}
+    </ul>
+<p>For more information about the changes included in this release, please visit the official 
+{{range $index, $version := .GoVersions}}
+<a href="{{$version.URL}}">Go {{$version.Version}}</a>{{if isLast $index $.GoVersions}}, and {{else}}, {{end}}
```

Comment:
This doesn't sound quite right... it generates this for me:

> <p>For more information about the changes included in this release, please visit the official <a href="https://go.dev/doc/devel/release#go1.22.3">Go 1.22.3</a>, <a href="https://go.dev/doc/devel/release#go1.21.10">Go 1.21.10</a>, and release notes.</p>

I would expect "visit the official Go 1.22.3 and Go 1.21.10 release notes". I was curious if there's some built-in, easy way to format this kind of list, and found https://stackoverflow.com/a/26901917, which isn't all that trivial.

Maybe we should make a table, instead? We could also include a link to the microsoft/go tag, which I haven't been doing before because it would be annoying to put together manually. 😄 It would be easy for a generator though.

Something like this, maybe:

<table>
<tr>
<th>Microsoft release</th>
<th>Upstream tag</th>
<tr>
<td><a href="https://github.com/microsoft/go/releases/tag/v1.22.3-1">v1.22.3-1</a></td>
<td>go1.22.3 <a href="https://go.dev/doc/devel/release#go1.22.3">release notes</a></td>
</tr>
<tr>
<td>v1.21.10-1</td>
<td>go1.21.10 release notes</td>
</tr>
</table>

Reply:
+1 on this. A table seems like the best way to format this data